src/meta/assessment/node.ts

```diff
@@ -6,6 +6,7 @@ export interface NodeValue {
   estimated?: boolean
   calculated?: boolean
   odp?: boolean
```

Comment:
do we still need odp boolean prop now that there's the odpId ?  should odpId be number ?

Reply:
let's just make odpId number . keep the odp property